spec/features/activities_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.feature 'Activities', type: :feature do
+  let(:user) { create(:user) }
+  let(:category) { create(:category, user:) }
+
+  scenario 'User views activities list for a category' do
+    activity1 = create(:activity, author: user, categories: [category])
+    activity2 = create(:activity, author: user, categories: [category])
+
+    login_as(user)
+
+    visit category_activities_path(category_id: category.id)
+
+    expect(page).to have_content(activity1.name)
+    expect(page).to have_content(activity1.amount)
+    expect(page).to have_content(activity2.name)
+    expect(page).to have_content(activity2.amount)
+  end
+
+  scenario 'Users sees an empty list message' do
+    login_as(user)
+
+    visit category_activities_path(category_id: category.id)
+
```

Comment:
- [ ] Kindly fix your tests as some are failing.

![image](https://github.com/Estete9/where-s-the-money/assets/81932478/aaeba9be-d7c1-4b23-b6af-0c83614c17bb)